src/main/kotlin/com/terraformation/backend/customer/api/GlobalRolesController.kt

```diff
@@ -0,0 +1,73 @@
+package com.terraformation.backend.customer.api
+
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.SimpleSuccessResponsePayload
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.customer.db.UserStore
+import com.terraformation.backend.customer.model.IndividualUser
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.default_schema.GlobalRole
+import com.terraformation.backend.db.default_schema.UserId
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1")
+@RestController
+class GlobalRolesController(
+    private val userStore: UserStore,
+) {
+  @ApiResponse200
+  @GetMapping("/globalRoles/users")
+  @Operation(summary = "Gets the list of users that have global roles.")
+  fun listGlobalRoles(): GlobalRoleUsersListResponsePayload {
+    requirePermissions { readGlobalRoles() }
```

Comment:
Would it make sense for this permission check to live in `UserStore.fetchWithGlobalRoles()`?

Reply:
Yes I think that makes more sense, updated.